.github/workflows/release.yml

```
@@ -38,17 +38,22 @@ jobs:
         with:
           tag-template: "{VERSION}"
           version-command: |
-            cat current-version.txt
+            cat galaxy.yml|grep version:|cut -d' ' -f2
 
       - name: Install Ansible
         run: |
           pip install --constraint=.github/workflows/constraints.txt ansible
 
-      - name: Trigger a new import on Galaxy.
+      - name: Release to Ansible Galaxy.
+        env:
+          ANSIBLE_GALAXY_TOKEN: ${{ secrets.GALAXY_TOKEN }}
         run: >-
-          ansible-galaxy role import
-          --api-key ${{ secrets.GALAXY_TOKEN }} $(echo ${{ github.repository }}
-          | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2)
+          ansible-playbook -i 'localhost,' galaxy-deploy.yml
+          -e "github_tag=${{ github.ref }}"
+          -e "namespace=$(echo ${{ github.repository }}
+          | cut -d/ -f1)"
+          -e "collection=$(echo ${{ github.repository }}
+          | cut -d/ -f2)"
```

looks like we need mode output for debugging
```suggestion
          ansible-playbook -D -vv -i 'localhost,' galaxy-deploy.yml
          -e "github_tag=${{ github.ref }}"
          -e "namespace=$(echo ${{ github.repository }}
          | cut -d/ -f1)"
          -e "collection=$(echo ${{ github.repository }}
          | cut -d/ -f2)"
```